src/audio/base_fw_intel.c

```diff
@@ -47,6 +47,10 @@ struct ipc4_modules_info {
 #define PLATFORM_LPSRAM_EBB_COUNT	(DT_REG_SIZE(DT_NODELABEL(sram1)) / SRAM_BANK_SIZE)
 #define PLATFORM_HPSRAM_EBB_COUNT	(DT_REG_SIZE(DT_NODELABEL(sram0)) / SRAM_BANK_SIZE)
 
+#define DT_NUM_SSP_BASE		DT_NUM_INST_STATUS_OKAY(intel_ssp)
```

Comment:
Whats happens if status not okay in DT ? Do we need a DT_xxxxx_OR() usage here to give a meaningful value if SSP is not enabled ?

Reply:
This the NUM variant, so it will return 0 if no matches found.